File: Occon.Auction.Server/Occon.Auction.Server.Api/Models/AuctionModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Occon.Auction.Server.Api.Models
{
    public class AuctionModel
    {
        public Guid Key { get; set; }
        public DateTime EndTime { get; set; }
        public string Deadline { get; set; }
        public List<Bidder> Bidders { get; set; }
        public string HighestBidder { get; set; }
        public decimal HighestBid { get; set; }
        public string BalanceTime { get; set; }
    }

    public class Bidder
    {
        public string Name { get; set; }
        public decimal Price { get; set; }
    }
}

File: Occon.Auction.Client/src/app/models/auction.model.ts
import { BidderModel } from './bidder.model';

export interface AuctionModel{
    EndTime: string,
    deadline: string,
    bidders: BidderModel[],
    highestBidder: string,
    highestBid: number,
    balanceTime: string,
    key: string
}